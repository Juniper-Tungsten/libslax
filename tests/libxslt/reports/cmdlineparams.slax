/* Machine Crafted with Care (tm) by slaxWriter */
version 1.1;

var $test = "no value";
param $test2 = "no value";

match / {
    <root> {
        expr "\n  test: ";
        expr $test;
        expr "\n  test2: ";
        expr $test2;
    }
}
