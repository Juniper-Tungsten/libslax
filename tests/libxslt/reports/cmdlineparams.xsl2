<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
  <!-- Machine Crafted with Care (tm) by slaxWriter -->
  <xsl:variable name="test" select="&quot;no value&quot;"/>
  <xsl:param name="test2" select="&quot;no value&quot;"/>
  <xsl:template match="/">
    <root>
      <xsl:text>
  test: </xsl:text>
      <xsl:value-of select="$test"/>
      <xsl:text>
  test2: </xsl:text>
      <xsl:value-of select="$test2"/>
    </root>
  </xsl:template>
</xsl:stylesheet>
