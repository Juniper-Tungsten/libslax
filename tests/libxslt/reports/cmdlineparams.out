<?xml version="1.0"?>
<root>
  test: no value
  test2: no value</root>
